src/test/resources/blockdevices.pan

```diff
@@ -69,8 +69,15 @@ unique template blockdevices;
             "size", 800,
             "volume_group", "vg0"
             ),
-        )
-	);
+        ),
+    "vxvm", nlist (
+        "vcslab.local", nlist("gnr.0", nlist (
+            "dev_path", "/dev/vx/dsk/vcslab.local/gnr.0",
```

Comment:
are `dev_path`, `disk_group` and `volume` reqlly required? seems redundant data given the keys? (i.e.
```pan
"vxvm", dict(
    "vcslab.local", dict(
         "gnr.0", dict(),
        ),
),
```
might be sufficient to derive these)